os-server/src/main/java/app/openschool/course/api/mapper/CourseMapper.java

```diff
@@ -69,11 +74,41 @@ public static CourseInfoDto toCourseInfoDto(Course course) {
   public static EnrolledCourse toEnrolledCourse(Course course, User user) {
     EnrolledCourse enrolledCourse =
         new EnrolledCourse(LocalDate.now(), course, user, CourseStatus.inProgress());
-    Set<EnrolledModule> enrolledModules = ModuleMapper.toEnrolledModules(course, enrolledCourse);
+    Set<EnrolledModule> enrolledModules =
+        EnrolledModuleMapper.toEnrolledModules(course, enrolledCourse);
     enrolledCourse.setEnrolledModules(enrolledModules);
     return enrolledCourse;
   }
 
+  public static Course toCourse(CreateCourseRequest request, User mentor) {
+    Course course = new Course();
+    course.setTitle(request.getTitle());
+    course.setDescription(request.getDescription());
+    course.setGoal(request.getGoal());
+    Category category = new Category();
+    category.setId(request.getCategoryId());
+    course.setCategory(category);
+    Difficulty difficulty = new Difficulty();
+    difficulty.setId(request.getDifficultyId());
+    course.setDifficulty(difficulty);
+    Language language = new Language();
+    language.setId(request.getLanguageId());
+    course.setLanguage(language);
+    course.setMentor(mentor);
+    Set<Keyword> keywords =
+        request.getKeywordIds().stream()
+            .map(
+                requestKeyword -> {
+                  Keyword keyword = new Keyword();
```

Comment:
please create constructor that takes id, so you will not need 3 line for mapping